pkg/controller/directvolumemigration/rsync.go

```diff
@@ -708,6 +781,15 @@ func (t *Task) deleteRsyncResources() error {
 		return err
 	}
 
+	if !t.Owner.Spec.DeleteProgressReportingCRs {
+		return nil
+	}
+
+	err = t.deleteProgressReportingDRs(destClient)
```

Comment:
Why `deleteProgressReportingDRs`? Shouldn't it be `CRs`?

Reply:
yes, I must be in a really weird state to make that kind of a typo HAHA

Reply:
done